### ISSUE TITLE
python -m ensurepip fails

### ISSUE THREAD

**Issue by rkuska** [fedora-python/rewheel#2]
```
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/lib64/python2.7/ensurepip/__main__.py", line 4, in <module>
    ensurepip._main()
  File "/usr/lib64/python2.7/ensurepip/__init__.py", line 248, in _main
    default_pip=args.default_pip,
  File "/usr/lib64/python2.7/ensurepip/__init__.py", line 102, in bootstrap
    dep_records = rewheel.find_system_records([p[0] for p in _PROJECTS])
  File "/usr/lib64/python2.7/ensurepip/rewheel/__init__.py", line 39, in find_system_records
    sys_sitepack = site.getsitepackages([sys.base_prefix, sys.base_exec_prefix])
AttributeError: 'module' object has no attribute 'base_prefix'
```

Neither `sys.base_prefix` nor `sys.base_exec_prefix` are present in python 2.7.

**Comment by rkuska** [comment]
```
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/lib64/python2.7/ensurepip/__main__.py", line 4, in <module>
    ensurepip._main()
  File "/usr/lib64/python2.7/ensurepip/__init__.py", line 248, in _main
    default_pip=args.default_pip,
  File "/usr/lib64/python2.7/ensurepip/__init__.py", line 102, in bootstrap
    dep_records = rewheel.find_system_records([p[0] for p in _PROJECTS])
  File "/usr/lib64/python2.7/ensurepip/rewheel/__init__.py", line 42, in find_system_records
    sys_sitepack = site.getsitepackages([sys.prefix, sys.exec_prefix])
TypeError: getsitepackages() takes no arguments (1 given)
```


This one should cause problems also in python3:
```
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/lib64/python2.7/ensurepip/__main__.py", line 4, in <module>
    ensurepip._main()
  File "/usr/lib64/python2.7/ensurepip/__init__.py", line 248, in _main
    default_pip=args.default_pip,
  File "/usr/lib64/python2.7/ensurepip/__init__.py", line 129, in bootstrap
    shutil.copy(whl, tmpdir)
  File "/usr/lib64/python2.7/shutil.py", line 119, in copy
    copyfile(src, dst)
  File "/usr/lib64/python2.7/shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: '/usr/lib64/python2.7/ensurepip/_bundled/setuptools-7.0-py2.py3-none-any.whl'
```

**Comment by rkuska** [comment]
More or less fixed by 6162cdaf but I still keep this open because of pypa/virtualenv#732
